pkg/common/scheduler/scheduler.go

```diff
@@ -35,6 +35,7 @@ type VPodLister func() ([]VPod, error)
 type Scheduler interface {
 	// Schedule computes the new set of placements for vpod.
 	Schedule(vpod VPod) ([]duckv1alpha1.Placement, error)
+	ScheduleVPod(vpod VPod) ([]duckv1alpha1.Placement, error)
```

Comment:
Let's keep the interface minimal. No need to expose ScheduleVPod

Reply:
I tried to change `ScheduleVPod` to `scheduleVPod` but there is compiler error complaining cannot use `StatefulSetScheduler` as interface Scheduler. I think it has something to do with `StatefulSetScheduler` and Scheduler interface being defined in different package so we need to make `scheduleVPod` exported.

Reply:
I suggest to remove `ScheduleVPod` altogether from the interface.

Reply:
If we remove ScheduleVPod, how do you propose to sort the placements?